lib/internal/test_runner/tap_lexer.js

```diff
@@ -468,11 +468,19 @@ class TapLexer {
   }
 
   #isLiteralSymbol(char) {
-    return (
-      (char >= 'a' && char <= 'z') ||
-      (char >= 'A' && char <= 'Z') ||
-      this.#isSpecialCharacterSymbol(char)
-    );
+    if (char.charCodeAt(0) > 127) {
+      return true;
+    }
+
+    if (!Number.isNaN(Number(char))) {
```

Comment:
Casting is a costly operation. If you are checking if char is a number or not:

`if (char >= '0' && char <= 9)`